### PR TITLE
GEOPY-358: fix script to start app in dev env [WIP]

### PR DESCRIPTION
**Pull request by sebhmg**
**GEOPY-358 - use local editable geoh5py in geoapps dev environment**

### CODE REVIEW

File: devtools/start-app.bat [hunk 1]
```diff
@@ -7,6 +7,7 @@ if !errorlevel! neq 0 (
   exit /B !errorlevel!
 )
 
+set PYTHONPATH=%project_dir%;%PYTHONPATH%
```
Comment (sebhmg) [inline devtools/start-app.bat:10]: from PyCharm terminal, `import geoapps` is working because `geoapps` is in the current folder...

**Review by andrewg-mira — APPROVED** [review]

File: devtools/start-app.bat (hunk 1 above)
Comment (benk-mira) [reply devtools/start-app.bat:10]: Thanks Seb, that last one did the trick.  Tested by printing a line in geoh5 init then running devtools/start-app, run first cell of inversion app and getting a print statement (nothing added to environment variables).

**Review by benk-mira — APPROVED** [review]
Works as expected now.  Thanks for the start-app script, too!

**Comment by sebhmg** [comment]
> Works as expected now. Thanks for the start-app script, too!

@benk-mira , thanks for the test.
I have pushed yet another change (installing geoapps editble in the local conda env, same as done for a user installation).
I think this way is better as you will also be able to have it working from PyCharm terminal, typing there: `jupyter notebook geoapps\applications\Index.ipynb`

**Comment by benk-mira** [comment]
image

**Review by benk-mira — CHANGES_REQUESTED** [review]

**Comment by sebhmg** [comment]
will create a new PR

**Pull request closed without merging**